packages/main/src/TableDragAndDrop.ts

```diff
@@ -12,7 +12,14 @@ export default class TableDragAndDrop extends TableExtension {
 	constructor(table: Table) {
 		super();
 		this._table = table;
-		DragRegistry.subscribe(this._table); // TODO: Where unsubscribe?
+	}
+
+	_ondragstart(e: DragEvent) {
```

Comment:
I'm not sure if there's a technical reason for it, but it feels really wrong: does every component that might become movable have to handle those things itself

Reply:
Hello @aborjinik, 

_"I'd recommend inspecting composedPath() to find the first UI5 element that is movable."_

This can work for the list and the table, but it is not valid for the ui5-tabcontainer. In the case of the TabContainer, the dragged element is not the element that is clicked and it is not present in the  composedPath().

`this._setDraggedElement!((e.target as TabInStrip).realTabReference);`
or 
`this._setDraggedElement!((e.target as TabInOverflow).realTabReference);`